### ISSUE TITLE
As alternative, should give Sourceforge a chance again?

### ISSUE THREAD

**Issue by MarkoCen** [upend/IF_MS_BUYS_GITHUB_IMMA_OUT#65]
Here is today's Sourceforge main page with this [GitHub importer](https://sourceforge.net/p/forge/documentation/GitHub%20Importer/)
![Sourceforge website screenshot](https://i.imgur.com/HBLpflv.png)

**Comment by Ominousness** [comment]
Hell no. Malware in installers? - tyvm.

**Comment by necrophcodr** [comment]
Sourceforge isn't even a free or open source, and it has a history of infrastructure problems, malware injections, and worse.

**Comment by Atavic** [comment]
It had a blackout in early 2018. Many projects landed on Github.

**Comment by loganabbott** [comment]
Hi all, president of SourceForge here. Just in case people don’t know, a lot has changed at SourceForge since my company acquired them in 2016. We [covered the improvements](https://sourceforge.net/blog/congratulations-github/) again here yesterday, but here is a summary since we took over.

The very first thing we did after acquiring SourceForge two years ago was remove bundled installers from projects, and vow that it'll never happen on SourceForge again. Since then, further improvements have included:

- Implemented malware scans with a partnership with Bitdefender for every single project on SourceForge
- Removed ads for developers (if you’re logged in, you’ll never see ads)
- Added HTTPS support for project website hosting and all downloads
- Completely redesigned SourceForge.net from the ground up

We'd love for people to give SourceForge another shot. We're committed to making SourceForge a solid, trustworthy place for FOSS hosting. If you're interested in giving us a shot, we have a [GitHub to SourceForge importer here](https://sourceforge.net/p/forge/documentation/GitHub%20Importer/).

**Comment by Bengt** [comment]
Thanks for improving Sourceforge. However, by comparison with GitLab's monthly (!) releases, these changes look like bringing a knife to a gunfight:

https://about.gitlab.com/2018/03/22/gitlab-10-6-released/
https://about.gitlab.com/2018/04/22/gitlab-10-7-released/
https://about.gitlab.com/2018/05/22/gitlab-10-8-released/

**Comment by Revivify** [comment]
Sourceforge is garbage. start a 501c3 to host a community developed alternative.
You'll get bought just like GitHub did. It's the dream of sillyvalley

**Comment by loganabbott** [comment]
I don't look at it as a zero sum game where winner takes all. We have over a million daily users and 500,000 projects hosted with us, so we're just focused on improving their experience.

We're actually profitable at SourceForge already with no plans to sell. We're not located in Silicon Valley, and we have taken no outside funding. That's not the case with GitLab and others. They're great products too, but I'd imagine they'll end up being bought by a company like Microsoft eventually, or will have to change drastically change their business model to be able to exist indefinitely.

At the end of the day, the more viable options out there for FOSS, the better. GitLab focuses on developers while SourceForge focuses on search and discovery for end-users where they can easily find and install the software they need, whether or not they're technical. It's apples and oranges and I think we can both co-exist.

**Comment by TwinProduction** [comment]
@Bengt on the other hand, GitLab has the annoying "This site uses cookies" pop-up...

**Comment by h3** [comment]
@loganabbott the most important difference to me is that even if Gitlab gets bought or change philosophy significantly, its source code can always be forked and maintained openly. No closed source solution can offer that level of flexibility and long-term security.

Incidentally, Gitlab knows very well that if they take a shit in the punch bowl, someone else is likely to quickly roll-out an alternative solution based on their very own source code.

https://gitlab.com/gitlab-org/gitlab-ce/blob/master/LICENSE

**Comment by rubo77** [comment]
@loganabbott: why don't you make the Source Code of your system open source as well?

**Comment by loganabbott** [comment]
@rubo77 SourceForge is built with [Apache Allura](https://allura.apache.org/), which is already open source.

**Comment by rubo77** [comment]
Are there many modifications from the standard Allura platform in sourceforge?

**Comment by loganabbott** [comment]
Yeah there are some, but when we make changes and improvements to the development tools on SourceForge, that code makes it's way back into Apache Allura, as our devs are the biggest contributors to the Allura project.

**Comment by williamknauss** [comment]
I'm going to weight in here. I don't believe Github is going down the drain (in regards to MS acquiring them). However in the spirit of open source, collaborating with others, and knowledge sharing I'd be willing to try Source Forge a try again. @loganabbott I'm willing to give SF a try again. BTW got any PHP jobs open there? :smiley:

**Comment by vassudanagunta** [comment]
Hey guys. I love the work you've been doing. Not everyone loves mine. Please read Issue 114.

**Comment by vassudanagunta** [comment]
Hello,

The GitHub Evacuation Project has moved to GitLab (not an endorsement or even a final home). Your enthusiasm and contribution is still needed. Please check out [the new project home](https://gitlab.com/upend/github), and read [the project wiki](https://gitlab.com/upend/github/evac-guide/wikis) for info to get restarted.

Thank you!

vas
